src/components/LiquidityMiningCampaigns/RewardsList/index.tsx

```diff
@@ -40,15 +40,15 @@ export function RewardsList({ dataFilter, pair, setDataFiler, loading }: Rewards
               itemsAmount={miningCampaigns.active.length}
               badgeTheme="orange"
             >
-              Campaigns
+              <div data-testid="active-campaigns" >Campaigns</div> 
```

Comment:
Already tried and doesn't work if I put in TabTitleComponent, this was the only way.
Otherwise we avoid adding extra div except in cases when we really have to.

Reply:
Preview deploy fixed